### PR TITLE
refactor: move styles closer to the components

### PR DESCRIPTION
**Pull request by chmelevskij**
Makes it a bit easier to see what styles are applied.

### CODE REVIEW

File: src/css/dark-theme.css
Comment (McGiverGim) [inline src/css/dark-theme.css:49]: This implies that dark theme now is not working?

**Review by McGiverGim — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by Asizon — APPROVED** [review]

**Comment by Asizon** [comment]
@chmelevskij i have merged it but it have some issues in the status bar, can you look here please?

**Pull request merged**
